backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  elevator                Boolean?
+  wheelchairRamp          Boolean?  @map("wheelchair_ramp")
+  serviceAnimalsAllowed   Boolean?  @map("service_animals_allowed")
+  accessibleParking       Boolean?  @map("accessible_parking")
+  parkingOnSite           Boolean?  @map("parking_on_site")
+  inUnitWasherDryer       Boolean?  @map("in_unit_washer_dryer")
+  laundryInBuilding       Boolean?  @map("laundry_in_building")
+  barrierFreeEntrance     Boolean?  @map("barrier_free_entrance")
+  rollInShower            Boolean?  @map("roll_in_shower")
+  grabBars                Boolean?  @map("grab_bars")
+  heatingInUnit           Boolean?  @map("heating_in_unit")
+  acInUnit                Boolean?  @map("ac_in_unit")
+  hearing                 Boolean?
+  visual                  Boolean?
+  mobility                Boolean?
+  barrierFreeUnitEntrance Boolean?  @map("barrier_free_unit_entrance")
+  loweredLightSwitch      Boolean?  @map("lowered_light_switch")
+  barrierFreeBathroom     Boolean?  @map("barrier_free_bathroom")
+  wideDoorways            Boolean?  @map("wide_doorways")
+  loweredCabinets         Boolean?  @map("lowered_cabinets")
+  listings                listings?
+
+  @@map("listing_features")
+}
+
+model ListingImages {
+  ordinal   Int?
+  listingId String   @map("listing_id") @db.Uuid
+  imageId   String   @map("image_id") @db.Uuid
+  assets    Assets   @relation(fields: [imageId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings  listings @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@id([listingId, imageId])
+  @@index([listingId])
+  @@map("listing_images")
+}
+
+model ListingMultiselectQuestions {
+  ordinal               Int?
+  listingId             String               @map("listing_id") @db.Uuid
+  multiselectQuestionId String               @map("multiselect_question_id") @db.Uuid
+  multiselectQuestions  MultiselectQuestions @relation(fields: [multiselectQuestionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings              listings             @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@id([listingId, multiselectQuestionId])
+  @@map("listing_multiselect_questions")
+}
+
+model ListingUtilities {
+  id          String    @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt   DateTime  @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt   DateTime  @updatedAt @map("updated_at") @db.Timestamp(6)
+  water       Boolean?
+  gas         Boolean?
+  trash       Boolean?
+  sewer       Boolean?
+  electricity Boolean?
+  cable       Boolean?
+  phone       Boolean?
+  internet    Boolean?
+  listings    listings?
+
+  @@map("listing_utilities")
+}
+
+// Note: missing [referralApplication, applicationConfig, showWaitlist, unitsSummarized] virtual property
+// Note: [reserved_community_description, result_link] formerly max length 4096;
+model listings {
```

Comment:
likely singular, other ones seem to be capitalized

Reply:
convention convo

but also good 👀 it should be capitalized camel case, will fix!